server/db/schema.ts

```diff
@@ -31,6 +31,13 @@ const ClassSchema = new Schema<ClassDocument>({
   classRating: { type: Number }, // the average class rating from reviews
   classWorkload: { type: Number }, // the average workload rating from reviews
   classDifficulty: { type: Number }, // the average difficulty rating from reviews
+  classSummary: { type: String, required: false }, // 50 word summary of the course generated by openAI
+  classTags: {
+    type: Map, // dictionary keys are lectures, assignments, professors, skills, and resources
+    of: [{ type: String }, { type: String }], // tuple where first element is the adjective and second element is its connotation
+    required: false
+  }, // tags describing lectures, assignments, professors, skills, and resources
+  freshness: { type: Number, default: 0 } // number of reviews since last openAI call
```

Comment:
I'm not sure if it would make sense to put all of this within an AI object instead? Like

```
AI: 
{
classSummary
classTags
freshness
}
```

or maybe renaming `classTags` to `classTagsReviews` will help differentiate between it and the tags for similar courses